src/card/basic/card.html

```diff
@@ -1,15 +1,15 @@
 <div ng-class="$ctrl.showTopBorder === 'true' ? 'card-pf card-pf-accented' : 'card-pf'">
   <div ng-if="$ctrl.showHeader()" ng-class="$ctrl.shouldShowTitlesSeparator ? 'card-pf-heading' : 'card-pf-heading-no-bottom'">
-    <div ng-if="$ctrl.showFilterInHeader()" ng-include="'card/basic/card-filter.html'"></div>
+    <div ng-class="{'hide': $ctrl.showSpinner === true}" ng-if="$ctrl.showFilterInHeader()" ng-include="'card/basic/card-filter.html'"></div>
```

Comment:
ng-show rather than adding the 'hide' class?

Reply:
Seems to be causing a lag with ng-if and ng-show

Reply:
Should be one or the other.   I did have to change ng-ifs to ng-shows in order to not break OpenShift e2e/Protractor tests, so I'd go with ng-show.